### PR TITLE
fixed typo, noted BlockBridge being payware

### PR DESCRIPTION
**Pull request by nizedk**
I don't know if the information section should reflect this, but the mentioned article is not about using docker securely, it's about buying their product to secure their volumes, which you also have to pay for.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -665,11 +665,11 @@ Services to securely store your Docker images.
...
 - [Docker and SELinux](https://www.projectatomic.io/docs/docker-and-selinux/)
 - [Docker Secure Deployment Guidelines](https://github.com/GDSSecurity/Docker-Secure-Deployment-Guidelines)
-- [Docker Securitiy - Quick Reference](https://binarymist.io/publication/docker-security/)
+- [Docker Security - Quick Reference](https://binarymist.io/publication/docker-security/)
 - [Docker Security Cheat Sheet](https://github.com/konstruktoid/Docker/blob/master/Security/CheatSheet.adoc)
 - [Docker Security: Are Your Containers Tightly Secured to the Ship? SlideShare](https://fr.slideshare.net/MichaelBoelen/docker-security-are-your-containers-tightly-secured-to-the-ship)
 - [How CVE's are handled on Offical Docker Images](https://github.com/docker-library/official-images/issues/1448)
-- [Improving Docker Security with Authenticated Volumes](https://www.blockbridge.com/improving-docker-security-with-authenticated-volumes/)
+- [Improving Docker Security with Authenticated Volumes](https://www.blockbridge.com/improving-docker-security-with-authenticated-volumes/) :heavy_dollar_sign:
```
Comment (veggiemonk) [inline README.md]: Indeed as @agebhar1 mentionned, this entry can be removed.

**Comment by agebhar1** [comment]
ping @nizedk

File: README.md (hunk 1 above)
Comment (nizedk) [reply README.md]: OK by me. Should I do something?

Comment (veggiemonk) [reply README.md]: @nizedk you can remove this entry or let us know and we'll do it

**Pull request merged**